Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.16-alpine
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
+
+COPY lz77/*.go ./lz77/
+COPY *.go ./
```

Comment:
Why not simply `ADD . .` instead of the four individual `COPY` commands?

Reply:
fixed